WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -23,41 +21,34 @@
 
 import java.util.List;
 
-public class PeopleManagementActivity extends AppCompatActivity {
-
-    private PeopleAdapter mPeopleAdapter;
+public class PeopleManagementActivity extends AppCompatActivity implements PeopleListFragment.OnPersonSelectedListener {
+    private static final String KEY_PEOPLE_LIST_FRAGMENT = "people-list-fragment";
+    private static final String KEY_PERSON_DETAIL_FRAGMENT = "person-detail-fragment";
```

Comment:
You're right those 2 are different, but maybe I should have used all-lowercase on the other one looking at all the other files:

https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/prefs/AccountSettingsActivity.java#L12
https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java#L61-L73
https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsActivity.java#L14-L15
https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsActivity.java#L38

Looking at the [style guidelines we use](https://source.android.com/source/code-style.html) I don't see any mention of the right side expression of constant declarations. So, maybe let's go with the rest of the app and use all lowercase instead. Sounds good?